api/v1beta1/event.proto

```diff
@@ -9,9 +9,597 @@ import "google/protobuf/wrappers.proto";
+    security_inode_unlink = 732;
+    security_socket_create = 733;
+    security_socket_listen = 734;
+    security_socket_connect = 735;
+    security_socket_accept = 736;
+    security_socket_bind = 737;
+    security_socket_setsockopt = 738;
+    security_sb_mount = 739;
+    security_bpf = 740;
+    security_bpf_map = 741;
+    security_kernel_read_file = 742;
+    security_inode_mknod = 743;
+    security_post_read_file = 744;
+    security_inode_symlink_event_id = 745;
+    security_mmap_file = 746;
+    security_file_mprotect = 747;
+    socket_dup = 748;
+    hidden_inodes = 749;
+    kernel_write = 750;
+    proc_create = 751;
+    kprobe_attach = 752;
+    call_usermode_helper = 753;
+    dirty_pipe_splice = 754;
+    debugfs_create_file = 755;
+    syscall_table_check = 756;
+    debugfs_create_dir = 757;
+    device_add = 758;
+    register_chrdev = 759;
+    shared_object_loaded = 760;
+    do_init_module = 761;
+    socket_accept = 762;
+    load_elf_phdrs = 763;
+    hooked_proc_fops = 764;
+    print_net_seq_ops = 765;
+    task_rename = 766;
+    security_inode_rename = 767;
+    do_sigaction = 768;
+    bpf_attach = 769;
+    kallsyms_lookup_name = 770;
+    do_mmap = 771;
+    print_mem_dump = 772;
+    vfs_utimes = 773;
+    do_truncate = 774;
+    file_modification = 775;
+    inotify_watch = 776;
+    security_bpf_prog = 777;
+    process_execute_failed = 778;
+    security_path_notify = 779;
+    hidden_kernel_module_seeker = 780;
+    module_load = 781;
+    module_free = 782;
+    execute_finished = 783;
+    security_bprm_creds_for_exec = 784;
+    max_common_id = 785;
+
+    // Events originated from user-space
+    net_packet_ipv4 = 2000;
+    net_packet_ipv6 = 2001;
+    net_packet_tcp = 2002;
+    net_packet_udp = 2003;
+    net_packet_icmp = 2004;
+    net_packet_icmpv6 = 2005;
+    net_packet_dns = 2006;
+    net_packet_dns_request = 2007;
+    net_packet_dns_response = 2008;
+    net_packet_http = 2009;
+    net_packet_http_request = 2010;
+    net_packet_http_response = 2011;
+    net_flow_end = 2012;
+    net_flow_tcp_begin = 2013;
+    net_flow_tcp_end = 2014;
+    max_user_net_id = 2015;
+    net_tcp_connect = 2016;
+    init_namespaces = 2017;
+    container_create = 2018;
+    container_remove = 2019;
+    existing_container = 2020;
+    hooked_syscall = 2021;
+    hooked_seq_ops = 2022;
+    symbols_loaded = 2023;
+    symbols_collision = 2024;
+    hidden_kernel_module = 2025;
+    ftrace_hook = 2026;
+    max_user_space = 2027;
+
+    // Capture meta-events
+    capture_file_write = 4000;
+    capture_exec = 4001;
+    capture_module = 4002;
+    capture_mem = 4003;
+    capture_pcap = 4004;
+    capture_net_packet = 4005;
+    capture_bpf = 4006;
+    capture_file_read = 4007;
+
+    // Signal meta-events
+    signal_cgroup_mkdir = 5000;
+    signal_cgroup_rmdir = 5001;
+    signal_sched_process_fork = 5002;
+    signal_sched_process_exec = 5003;
+    signal_sched_process_exit = 5004;
```

Comment:
I think none of these should be exposed to the user since they are all internal to tracee